src/App.jsx

```diff
@@ -1,26 +1,26 @@
-import React, { useEffect } from "react";
-import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+// import React, { useEffect } from "react";
```

Comment:
Leaving commented out code is against best practices. 
In cases like this, the right thing to do is to delete the code. 

You can always come back to the PR where the code is deleted to recover any code you need.